src/test/java/com/hivemq/cli/commands/hivemq/script/ScriptDeleteCommandTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019-present HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hivemq.cli.commands.hivemq.script;
+
+import com.hivemq.cli.commands.hivemq.datahub.OutputFormatter;
+import com.hivemq.cli.openapi.ApiException;
+import com.hivemq.cli.openapi.hivemq.DataHubScriptsApi;
+import com.hivemq.cli.rest.HiveMQRestService;
+import com.hivemq.cli.utils.TestLoggerUtils;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import picocli.CommandLine;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyDouble;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ScriptDeleteCommandTest {
+
+    private final @NotNull HiveMQRestService hiveMQRestService = mock();
+    private final @NotNull OutputFormatter outputFormatter = mock();
+    private final @NotNull DataHubScriptsApi scriptsApi = mock(DataHubScriptsApi.class);
```

Comment:
```suggestion
    private final @NotNull DataHubScriptsApi scriptsApi = mock();
```

Also in other classes